lib/webidl2.js

```diff
@@ -714,15 +718,15 @@
           consume(OTHER, ";") || error("Missing semicolon after interface");
           return ret;
         }
-        var ea = extended_attrs(store ? mems : null);
+        const ea = extended_attrs(store ? mems : null);
```

Comment:
nit: whitespace ?

Reply:
Be nice if we got rid of the `const x = {}, y = {}; ` etc. pattern usage and instead do "const x = {}; const y = {}" and so on. The "," is a pretty bad practice. We can do that in different PR tho.

Reply:
>nit: whitespace ?

Could you clarify more? `extended_attrs` already calls `all_ws` internally so no need to process whitespaces twice.

Reply:
sorry, I meant changing:

```
const mems = [],
  ret = {};
```

To:
```
const mems = [];
const ret = {};
```

Reply:
With regards to whitespace.... I literally meant that the indentation of your code is incorrect - just a nit :) The actual processing is  all good 👍